src/guiFormSpecMenu.h

```diff
@@ -199,10 +200,10 @@ class GUIFormSpecMenu : public GUIModalMenu
 		FieldSpec(const std::string &name, const std::wstring &label,
 				const std::wstring &fdeflt, int id) :
```

Comment:
Maybe change it to `default` so it's actually legible?  It'll also be consistent with the rest of the parameters which are otherwise identical to the members, but lacking the f prefix.